### ISSUE TITLE
FlatList on iphoneX/XS/XS PLUS with small font size cannot pull down

### ISSUE THREAD

**Issue by winfredxu** [facebook/react-native#23690]
## 🐛 Bug Report
I'm using FlatList to display the items, also added a RefreshControl with it to implement pull down refreshing. 
On iphone x/iphone xs/iphone xs plus, if I set the font size to smaller. (iphone-Settings-General-Accessibility-Larger Text, on the bottom, select the left 1, or 2 level font size), then the FlatList cannot be pulled down. 
And while pulling down, I saw that caused screen component re-rended. 
![1](https://user-images.githubusercontent.com/13715502/53562959-aec3b600-3b8d-11e9-9514-e3a838e7c9ce.jpeg)
![2](https://user-images.githubusercontent.com/13715502/53562978-bedb9580-3b8d-11e9-9603-ccf36697ebea.jpeg)

Other IOS devices or issue devices with larger font size don't have such issue.
![3](https://user-images.githubusercontent.com/13715502/53562990-c4d17680-3b8d-11e9-9654-ccc5fcd6145e.jpeg)

Remove the RefreshControl has the same result, so that means that's not the RefreshControl issue.
The ScrollView component also doesn't have this issue.

## To Reproduce
1. Implement FlatList as regular.
2. Change the font size of iphone through (iphone-Settings-General-Accessibility-Larger Text, on the bottom, select the left 1, or 2 level font size)
3. Back to FlastList just implemented

## Expected Behavior
The FlatList can be pulled down with IOS small font size.

## Code Example
<FlatList contentContainerStyle={styles.flatListContainerStyle}
            data={myItems}
            keyExtractor={(item, index) => index.toString()}
            renderItem={
              ({ item }) =>
                <Item>Examples</Item>
            }
            ListHeaderComponent={
              <View style={styles.separatorStyle} />
            }
            ItemSeparatorComponent={
              () => {
                return <View style={styles.separatorStyle} />
              }
            }
            onEndReachedThreshold={0.2}
            onEndReached={this.scorllLoading}
          />

## Environment
 React Native Environment Info:
    System:
      OS: macOS High Sierra 10.13.6
      CPU: (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
      Memory: 543.20 MB / 16.00 GB
      Shell: 5.3 - /bin/zsh
    Binaries:
      Node: 10.14.1 - /usr/local/bin/node
      npm: 6.8.0 - ~/.npm-global/bin/npm
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
      Android SDK:
        API Levels: 28
        Build Tools: 28.0.2, 28.0.3
        System Images: android-28 | Google APIs Intel x86 Atom
    IDEs:
      Android Studio: 3.3 AI-182.5107.16.33.5264788
      Xcode: 10.1/10B61 - /usr/bin/xcodebuild
    npmPackages:
      react: 16.6.3 => 16.6.3 
      react-native: 0.58.4 => 0.58.4 
    npmGlobalPackages:
      react-native-cli: 2.0.1
      react-native-git-upgrade: 0.2.7
      react-native: 0.57.8

**Comment by winfredxu** [comment]
> Can you run `react-native info` and edit your issue to include these results under the **Environment** section?
> 
> If you believe this information is irrelevant to the reported issue, you may write `[skip envinfo]` under **Environment** to let us know.

Updated! Thanks!

**Comment by winfredxu** [comment]
Mostly this is a react-navigation issue, only the FlatList is inside the navigatior created with 'createMaterialTopTabNavigator' will have this issue, and not only for FlatList, also for ScrollView. Since this may not related with react-native, close the issue.
